src/lib/protocols/tls.c

```diff
@@ -1432,6 +1432,11 @@ int processClientServerHello(struct ndpi_detection_module_struct *ndpi_struct,
 	      NDPI_SET_BIT(flow->risk, NDPI_TLS_NOT_CARRYING_HTTPS);
 	    }
 
+	    if(flow->protos.stun_ssl.ssl.encrypted_sni.esni &&
+	       flow->protos.stun_ssl.ssl.client_requested_server_name[0] != '\0') {
```

Comment:
indent - 1

Reply:
> indent - 1

Sorry but I don't see any indentation issue. For the sake of coherence, I tried to indent this if-block like the previous one: one tab + spaces.
What am I missing?

Reply:
Sorry, my fault. I'm not seeing the obvious.